console/src/channel/CreateCalculated.tsx

```diff
@@ -123,19 +158,19 @@ interface InternalProps extends Pick<RendererProps, "onClose"> {
 const Internal = ({ onClose, initialValues }: InternalProps): ReactElement => {
   const client = Synnax.use();
 
-  const methods = Form.use<typeof schema>({ schema, values: initialValues });
+  const methods = Form.use<typeof schema>({
+    schema,
+    values: initialValues,
+    sync: true,
+  });
 
   const addStatus = Status.useAggregator();
 
   const [createMore, setCreateMore] = useState(false);
   const { mutate, isPending } = useMutation({
     mutationFn: async (createMore: boolean) => {
       if (client == null) throw new Error("Client not available");
```

Comment:
we dont handle it like this in the original create channel window. we just return.